src/maconn.ts

```diff
@@ -1,33 +1,33 @@
-import {MultiaddrConnection, MultiaddrConnectionTimeline} from "@libp2p/interface-connection";
-import { logger } from '@libp2p/logger';
-import {Multiaddr} from "@multiformats/multiaddr";
-import {Source, Sink} from "it-stream-types";
-import {nopSink, nopSource} from "./util.js";
+import { MultiaddrConnection, MultiaddrConnectionTimeline } from '@libp2p/interface-connection'
+import { logger } from '@libp2p/logger'
+import { Multiaddr } from '@multiformats/multiaddr'
+import { Source, Sink } from 'it-stream-types'
+import { nopSink, nopSource } from './util'
```

Comment:
The linter did not request that, but I was able to get it working w/o the extensions, so went that way since it looks cleaner.  Do I need to add them back (note: only some imports had the extensions).

Reply:
I'll take a look at the dist creation process, seems like this shouldn't be happening.

Reply:
Did you run 1aegir release`?

Reply:
I found it was only required for imports from source files local to the package. It looks like the `js-libp2p` project retains the extensions for their imports from local package files, see their [`index.ts`](https://github.com/libp2p/js-libp2p/blob/master/src/index.ts) as an example.

Reply:
I ran `npx aegir release` on my local copy of this branch, and then attempted to build the demo source which imports the local branch copy. I still saw the same resolution error which even suggests adding the file extension:
```
Module not found: Error: Can't resolve './transport' in '/Users/ryanplauche/Development/LBL/Protocol/js-libp2p-webrtc/dist/src'
Did you mean 'transport.js'?
BREAKING CHANGE: The request './transport' failed to resolve only because it was resolved as fully specified
```

Reply:
Interesting, I thought adding the extensions was an anti-pattern since it's done implicitly.  I'll add them back.

Reply:
This is reverted.  I also separated out the src imports from the package imports and did some OCD reordering.